react/week1/todo-app/src/App.js

```diff
@@ -0,0 +1,81 @@
+import logo from './logo.svg';
+import './App.css';
+
+function DateItem(props) {
+  let d = new Date();
+  d.setFullYear(props.date.year, props.date.month - 1, props.date.day);
+  const today = new Date();
+  const validDate = (d > today) ? 'valid': 'expired';
+  return (
+    <p>{props.date.day}/ {props.date.month}/ {props.date.year} {`${validDate}`}</p>
+  )
+}
+
+function TodoItem(props) {
```

Comment:
this is perfectly fine. It is a little bit more idiomatic react to use arrow functions:

    const TodoItem = () => { ... }

but it's totally up to you.

Reply:
Thank you so much for your comments and feedback! I handled all comments and made changes to my code.

Reply:
Looks super good!